app/app/urls.py

```diff
@@ -514,6 +514,21 @@
         retail.emails.successful_contribution,
         name='admin_successful_contribution'
     ),
+    path(
+        '_administration/email/hackathon_ends/<str:hackathon>/<str:username>',
+        retail.emails.hackathon_ends,
+        name='hackathon_ends'
+    ),
+    path(
+        '_administration/email/bounty_not_submitted/<int:bounty>',
```

Comment:
Does this not need hackathon id ? Is that taken from bounty

Reply:
Taken from the bounty.